Sift/Sift.csproj

```diff
@@ -7,7 +7,7 @@
     <VersionPrefix>0.7.0</VersionPrefix>
     <VersionSuffix>beta</VersionSuffix>
     <PackageReleaseNotes>Beta release</PackageReleaseNotes>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
I think `README.md` needs to be updated. It currently says `The official Sift .NET client, supporting .NET Standard 2.0+`. According to these [.net docs](https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0), the minimum version required for standard 2.1 is 3.0.

Reply:
im looking at maybe using this package in future, and making this change would block us. We have an app still running on net48 that we haven't ported forward into net6 yet. There is probably a large group of users that are still on net48

Reply:
That's a good point.

@vineethKExalture I think we should remove this change because we don't appear to be using 2.1 functionality.

Reply:
@mpierotti-sift  Downgraded to 2.0 for the compatibility for existing old users